src/components/list-view/index.tsx

```diff
@@ -272,6 +279,20 @@ class ListView extends Component<Props, State> {
     const customFooterLoaded = showFooter && launchFooterLoaded && !hasMore; // 渲染renderLoadedText
     const footerLoading = showFooter && !launchFooterLoading && hasMore;
     const customFooterLoading = showFooter && launchFooterLoading && hasMore; // 渲染renderNoMore
+
+    const isScrollingRequired = this.targetScrollTop !== this.state.scrollTop && this.reachedTargetScrollTop === true;
+    const isDeltaYChanged = scrollDeltaY !== this.prevScrollDeltaY;
+
+    if (scrollDeltaY && (isScrollingRequired || isDeltaYChanged)) {
+      this.reachedTargetScrollTop = false;
+
+      const revisiting = this.targetScrollTop === this.state.scrollTop + scrollDeltaY;
```

Comment:
use `this.targetScrollTop === (this.state.scrollTop + scrollDeltaY);` has better readability